WPA3 Compatibility and STA AKM Telemetry

AmaleshNandh29 opened the pull request.

AmaleshNandh29 committed:
RDKB-58445 Add RSN Override for WPA3 Compatibility

Impacted Platforms:
Tech-XB7, Tech-XB8, CBRv2

Reason for change: Added RSNO in hostap-2.10 for WPA3-Compatibility mode

Test Procedure: Enable WPA3-Compatibility RFC and set security mode as
                WPA3-Personal-Compatibility.
                Check Client connectivity and internet

Risks: Medium
Priority:P1
Signed-off-by:Amalesh_Nandh@comcast.com

AmaleshNandh29 committed:
RDKB-58443:Add WPA3-Compatibility Telemetry Changes

Impacted Platforms:
XB10

Reason for change:
To add WPA3-Personal-Compatibility mode telemetry changes in rdk-wifi-hal,
onewifi,hostapd

Test Procedure:
Check Client connectivity and internet , respective telemetry logs

Risks: Medium
Priority:P1

Change-Id: If1986edd219753de6c6261769df87a90f45b2b17
Signed-off-by:Harshavardhan_Pulluru@comcast.com

AmaleshNandh29 committed:
RDKB-58443:Add WPA3-Compatibility RSNO Changes for 2.11

Impacted Platforms:
XB10

Reason for change:
To add WPA3-Personal-Compatibility RSNO changes in hostapd-2.11

Test Procedure: Enable WPA3-Compatibility RFC and set security mode as
                WPA3-Personal-Compatibility.
                Check Client connectivity and internet

Risks: Medium
Priority:P1

Signed-off-by:Amalesh_Nandh@comcast.com

AmaleshNandh29 committed:
RDKB-58940 Add RSN Override for WPA3 Compatibility

Impacted Platforms:
Tech-XB7, Tech-XB8, CBRv2

Reason for change: Added RSNO in hostap-2.10 for WPA3-Compatibility mode

Test Procedure: Enable WPA3-Compatibility RFC and set security mode as
                WPA3-Personal-Compatibility.
                Check Client connectivity and internet

Risks: Medium
Priority:P1
Signed-off-by:Amalesh_Nandh@comcast.com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/hostap-patches/7/rdkcentral/hostap-patches

* Commit: f2b0e8bef92e8b2f0e85755307ac7987b275f03f

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/8bb3b90d43800a0ec00ec7dabaac4e31)'

mhughesacn commented:
Please add a header to all the patches.  It is particularly important to give the source of the code in the patch.

gsathish86 reviewed (changes requested):
Please add header to the patch.

AmaleshNandh29 committed:
RDKB-58445 RDKB-58443 RDKB-58940 WPA3 Compatibility and STA AKM telemetry

Impacted Platforms:
Tech-XB7, Tech-XB8, Tech-XB10, CBRv2

Reason for change: Added RSNO in hostap-2.10 for WPA3-Compatibility mode

Test Procedure: Enable WPA3-Compatibility RFC and set security mode as
                WPA3-Personal-Compatibility.
                Check Client connectivity and internet

Risks: Medium
Priority:P1
Signed-off-by:Amalesh_Nandh@comcast.com
Signed-off-by:Harshavardhan_Pulluru@comcast.com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/hostap-patches/7/rdkcentral/hostap-patches

* Commit: ea5d3ac3c919649bec401251bf999ecee6c1812a

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/9a055b1e6b2cc7f85943db6ced06ef38)'

AmaleshNandh29 committed:
RDKB-58445 RDKB-58443 RDKB-58940 WPA3 Compatibility and STA AKM telemetry

Impacted Platforms:
Tech-XB7, Tech-XB8, Tech-XB10, CBRv2

Reason for change: Added RSNO in hostap-2.10 for WPA3-Compatibility mode

Test Procedure: Enable WPA3-Compatibility RFC and set security mode as
                WPA3-Personal-Compatibility.
                Check Client connectivity and internet

Risks: Medium
Priority:P1
Signed-off-by:Amalesh_Nandh@comcast.com
Signed-off-by:Harshavardhan_Pulluru@comcast.com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/hostap-patches/7/rdkcentral/hostap-patches

* Commit: ae492333d572721a44da494f0e1009691773a404

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/9f1efbf045dd95873683597c60fe8ba5)'

mhughesacn commented:
Hi Amalesh: Sorry if this is a dumb question but the headers give particular SHA1s as the source.  Are these SHA1s from an official hostap repo or somewhere in Comcast?  I tried git://w1.fi/hostap.git on one.  In other words, is this a backport from opensource land or Comcast own code?

AmaleshNandh29 commented:
@mhughesacn The commit ID in the last line of the Header is from Comcast Repo

mhughesacn commented:
Hi Amalesh: Please will you add an extra line to the patch headers saying "Source: Comcast" or "Source: Upstream from opensource" depending on whether the new or modified code is written by Comcast or taken from external hostap.

AmaleshNandh29 committed:
RDKB-58445 RDKB-58443 RDKB-58940 WPA3 Compatibility and STA AKM telemetry

Impacted Platforms:
Tech-XB7, Tech-XB8, Tech-XB10, CBRv2

Reason for change: Added RSNO in hostap-2.10 for WPA3-Compatibility mode

Test Procedure: Enable WPA3-Compatibility RFC and set security mode as
                WPA3-Personal-Compatibility.
                Check Client connectivity and internet

Risks: Medium
Priority:P1
Signed-off-by:Amalesh_Nandh@comcast.com
Signed-off-by:Harshavardhan_Pulluru@comcast.com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 4 files pending identification.
* Protex Server Path: /home/blackduck/github/hostap-patches/7/rdkcentral/hostap-patches

* Commit: d90aa019db098830fcb6382194ce21b8f73f4c72

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/b07b01ecec2311b1355a5252912fa90f)'
